docs/book/component-guide/orchestrators/lightning.md

```diff
@@ -0,0 +1,187 @@
+
+
+## How to use it
+
+To use the Lightning AI orchestrator, you need:
+
+*   The ZenML `lightning` integration installed. If you haven't done so, run
+
+    ```shell
+    zenml integration install lightning
+    ```
+
+* A [remote artifact store](../artifact-stores/artifact-stores.md) as part of your stack.
+
+
+* [Lightning AI credentials](#lightning-ai-credentials)
+
+
+### Lightning AI credentials
+
+* `LIGHTNING_USER_ID`: Your Lightning AI user ID
+
+* `LIGHTNING_API_KEY`: Your Lightning AI API key
+
+* `LIGHTNING_USERNAME`: Your Lightning AI username (optional)
+
+* `LIGHTNING_TEAMSPACE`: Your Lightning AI teamspace (optional)
+
+* `LIGHTNING_ORG`: Your Lightning AI organization (optional)
+
+
+Alternatively, you can configure these credentials when registering the orchestrator.
+
+
+We can then register the orchestrator and use it in our active stack:
+
+```shell
+zenml orchestrator register lightning_orchestrator \
+    --flavor=lightning \
+    --user_id=<YOUR_LIGHTNING_USER_ID> \
+    --api_key=<YOUR_LIGHTNING_API_KEY> \
+    --username=<YOUR_LIGHTNING_USERNAME> \
+    --teamspace=<YOUR_LIGHTNING_TEAMSPACE> \
+    --organization=<YOUR_LIGHTNING_ORGANIZATION>
+
+# Register and activate a stack with the new orchestrator
+zenml stack register lightning_stack -o lightning_orchestrator ... --set
+```
+
+You can also configure the orchestrator at pipeline level, using the `orchestrator` parameter.
+
+
+```python
+from zenml.integrations.lightning.flavors.lightning_orchestrator_flavor import LightningOrchestratorSettings
+
+
+lightning_settings = LightningOrchestratorSettings(
+    main_studio_name="my_studio",
+    machine_type="cpu",
+    async_mode=True,
+    custom_commands=["pip install -r requirements.txt", "do something else"]
+)
+
+@pipeline(
+    settings={
+        "orchestrator.lightning": lightning_settings
+    }
+)
+def my_pipeline():
+    ...
+```
+
+
+{% hint style="info" %}
+ZenML will build a wheel that includes all the files within you zenml repository and upload it to the lightning studio.
+For this reason you need make sure that you are running `zenml init` in the same directory where you are running your pipeline.
+{% endhint %}
+
+
+You can now run any ZenML pipeline using the Lightning AI orchestrator:
+
+```shell
+python file_that_runs_a_zenml_pipeline.py
+```
+
+
+### Lightning AI UI
+
+Lightning AI provides its own UI where you can monitor and manage your running applications, including the pipelines orchestrated by ZenML.
+For any runs executed on Lightning AI, you can get the URL to the Lightning AI UI in Python using the following code snippet:
+
+```python
+from zenml.client import Client
+
+pipeline_run = Client().get_pipeline_run("<PIPELINE_RUN_NAME>")
+orchestrator_url = pipeline_run.run_metadata["orchestrator_url"].value
+```
+
```

Comment:
i think we need a pragraph about how zenml spins up the studio, and what commands it runs. also we should talk about async mode and what happens then. finally we should talk about reuse of studios and the ability to even point to a base studio